forklift/services/elasticsearch.py

```diff
@@ -104,7 +104,11 @@ def check_available(self):
             url = replace_part(url, path='')
             es_response = urllib.request.urlopen(url.geturl())
             es_status = json.loads(es_response.read().decode())
-            if es_status['status'] != 200:
+            try:
```

Comment:
I feel like this is starting to get a little long, did you think about splitting the status check out into a new method?

What would you thins about inlining getting the status, too: `status = es_status.get('status', es_response.status)` ?